src/test/java/nl/tudelft/semgroup4/StartScreenStateTest.java

```diff
@@ -0,0 +1,226 @@
+        Mockito.when(mockedController.isAreaClicked(Mockito.any(), Mockito.any())).thenReturn(
+                false);
+        state.updateOptionsButton(mockedContainer, mockedSBG);
+        Mockito.verify(mockedSBG, Mockito.never()).enterState(States.OptionsState);
+    }
+
+    @Test
+    public void testUpdatePlayer1ButtonNoClick() throws SlickException {
+        Mockito.when(mockedController.isAreaClicked(Mockito.any(), Mockito.any())).thenReturn(
+                false);
+        state.updatePlayer1Button(mockedContainer, mockedSBG);
+        Mockito.verify(mockedSBG, Mockito.never()).enterState(States.GameState);
+    }
+
+    @Test
+    public void testUpdatePlayer2ButtonNoClick() throws SlickException {
+        Mockito.when(mockedController.isAreaClicked(Mockito.any(), Mockito.any())).thenReturn(
+                false);
+        state.updatePlayer2Button(mockedContainer, mockedSBG);
+        Mockito.verify(mockedSBG, Mockito.never()).enterState(States.GameState);
+    }
+
+    @Test
+    public void testUpdateExitButtonNoClick() throws SlickException {
+        Mockito.when(mockedController.isAreaClicked(Mockito.any(), Mockito.any())).thenReturn(
+                false);
+        state.updateExitButton(mockedContainer, mockedSBG);
+        Mockito.verify(mockedContainer, Mockito.never()).exit();
+    }
+
+    @Test
+    public void testUpdateHighscoresButtonClick() throws SlickException {
+        Mockito.when(mockedController.isAreaClicked(Mockito.any(), Mockito.any())).thenReturn(
+                true);
+        GameState mockedNewState = Mockito.mock(GameState.class);
+        Mockito.when(mockedSBG.getState(States.HighscoresState)).thenReturn(mockedNewState);
+
+        state.updateHighscoresButton(mockedContainer, mockedSBG);
+        Mockito.verify(mockedSBG).enterState(States.HighscoresState);
+    }
+
+    @Test
+    public void testUpdatePlayer1ButtonClick() throws SlickException {
+        Mockito.when(mockedController.isAreaClicked(Mockito.any(), Mockito.any())).thenReturn(
+                true);
+        GameState mockedNewState = Mockito.mock(GameState.class);
+        Mockito.when(mockedSBG.getState(States.GameState)).thenReturn(mockedNewState);
+
+        state.updatePlayer1Button(mockedContainer, mockedSBG);
+        Mockito.verify(mockedSBG).enterState(States.GameState);
+    }
+
+    @Test
+    public void testUpdatePlayer2ButtonClick() throws SlickException {
+        Mockito.when(mockedController.isAreaClicked(Mockito.any(), Mockito.any())).thenReturn(
+                true);
+        GameState mockedNewState = Mockito.mock(GameState.class);
+        Mockito.when(mockedSBG.getState(States.GameState)).thenReturn(mockedNewState);
+
+        state.updatePlayer2Button(mockedContainer, mockedSBG);
+        Mockito.verify(mockedSBG).enterState(States.GameState);
+    }
+
+    // Cannot be tested due to the following calls in the method declaration:
```

Comment:
No, I tried doing that. Somewhere in the method a dependency is used that crashes, which is different from the input.

< testing